scripts/templates/detector.tf.j2

```diff
@@ -49,6 +49,7 @@ resource "signalfx_detector" "{{ id }}" {
     {%- for key, signal in signals.items() -%}
       {%- if 'metric' in signal %}
     {{ key }} = data('{{ signal.metric }}', filter={%- if exclude_not_running_vm is defined and exclude_not_running_vm -%}${local.not_running_vm_filters} and {% endif -%}{%- if filtering is defined and filtering -%}{{ filtering_variable }} and {% endif -%}{%- if signal.filter is defined and signal.filter is string -%}{{ signal.filter }} and {% endif -%}${module.filtering.signalflow}{%- if signal.rollup is defined -%}, rollup='{{ signal.rollup }}'{% endif -%}{%- if signal.extrapolation is defined and signal.extrapolation -%}, extrapolation='{{ signal.extrapolation }}'{% endif -%})
+        {{- signal.function | default('') }}
```

Comment:
I am not fan of this because it is already possible using `formula` which is more powerful, seems more obvious and understandable and lead to separate lines in signalflow to improve the readibility.
also splitting into seperate signals allows the user to display the original metric in addition to the result signal easily whithout to change the program (just have to click on the eye) for debugging purpose